### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Luc-Siat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
+# Cup of Tickets
 
-Frontend URL: 
-https://orange-pond-040622003.2.azurestaticapps.net
+Project made by [Lidia Orfanu](https://github.com/LidiaOrfanu), [Luc Siat](https://github.com/Luc-Siat), [Donna Soujeh](https://github.com/proksimacentauri), [Caspar Rödöö](https://github.com/cparn) in a week and a half.
 
-Backend URL:
-https://cup-of-tickets-backend.azurewebsites.net
+[Frontend URL](https://orange-pond-040622003.2.azurestaticapps.net)
+[Backend URL](https://cup-of-tickets-backend.azurewebsites.net)
```
Comment (cparn) [inline README.md:7]: Change frontend to deployed app.
Either remove backend or change to swagger, just having the api url make no sense

File: README.md [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
 
-The website is ticketing app for developer (similar to Jira) and therefore the creation of account cannot be made by the user.
-In order to navigate the website please use those credentials which will grant you a mock admin access.
+Techstack:
+  - C# with Entity Framework
+  - React
```
Comment (cparn) [inline README.md:10]: Add Vite

**Review by cparn — CHANGES_REQUESTED** [review]
Some comments

File: README.md [hunk 3]
```diff
@@ -1,16 +1,27 @@
...
+  - TypeScript
```
[the remark this answers is not in the document]
Comment (Luc-Siat) [reply README.md:11]: might be a lot if I had vite there

File: README.md [hunk 4]
```diff
@@ -1,16 +1,27 @@
...
+Some Context:
 
-username : mathias.viklund@appliedtechnology.se
-password : Password123
+The website is a ticketing app for developer (similar to Jira), the main idea was to create an application that would have a role based authentication/authorization. The user will have different tabs available based on his role.
+The role part is being made through Auth0, the website used their tokens to exchange with the backend. The app being aimed toward companies we deactived the option to create and account.
+In order to navigate the website please use those credentials which will grant you a mock admin access.
 
+username : "mathias.viklund@appliedtechnology.se"
+password : "Password123"
```
[the remark this answers is not in the document]
Comment (Luc-Siat) [reply README.md:27]: oups tack!

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (cparn) [reply README.md:11]: Vite is apparently very valuable, in my interview they were low key impressed we used it. 🤣 Even though all we did was white-it-up

**Pull request merged**